extension/tasks/dependabotV2/utils/getSharedVariables.ts

```diff
@@ -160,6 +165,9 @@ export default function getSharedVariables(): ISharedVariables {
   let commentPullRequests: boolean = tl.getBoolInput('commentPullRequests', false);
   let abandonUnwantedPullRequests: boolean = tl.getBoolInput('abandonUnwantedPullRequests', true);
 
+  let dependabotCliImage: string | undefined = tl.getInput('dependabotCliImage');
```

Comment:
Don't the new inputs need to be declared in `task.json`?

Reply:
fixed